docs/software/cloud.md

```diff
@@ -527,4 +528,92 @@ Source code can be found at <https://github.com/mozilla-services/syncserver>.
 
 Credits: This software title has been added to DietPi-Software by CedArctic, many thanks! :D
 
+## Bitwarden_RS
+
+Bitwarden_RS is an unofficial Bitwarden password manager server with web interface, written in Rust.
+
+![Bitwarden_RS web vault screenshot](../assets/images/dietpi-software-cloud-bitwarden_rs.png)
+
+=== "First access"
+
+    - During install, a self-signed 4096-bit RSA TLS certificate is created to allow encrypted HTTPS access, which is required for access with most Bitwarden clients and reasonable as of the sensitivity of the data a password manager handles.
+    - Most web browsers will warn you on access that the certificate is not trusted, although usually you can choose to ignore that and still access the web vault.
+    - Most Bitwarden clients on the other hand will deny to access your server, as long as the certificate is not trusted.
+    - As fast as you have a public domain name for your DietPi server, we recommend to request an official trusted CA certificate, e.g. via `dietpi-letsencrypt` and setup either a reverse proxy, or configure Bitwarden_RS to use the retrieved key and certificate directly via ROCKET_TLS setting in the config file (see "Directories" tab).
+
+    #### How to add a self-signed certificate to the OS' Trusted Root Certification Authorities store
+
+    === "Windows 10"
+
+        1. In your browser, next to the address bar, select the warning or lock icon.
+            Select the certificate button to open Windows' Certificate view.
+        2. Switch to the "Details" tab.
+            ![Import certificate on Windows 10, screenshot 1](../assets/images/import_cert_windows_1.png)
+        3. Select "Save to file".
```

Comment:
I would propose to add all the screenshots in this tab using a width of 600 px only. I feel this is a bit more harmonic in size.  
`![Import certificate on Windows 10, screenshot 1](../assets/images/import_cert_windows_1.png){: style="width:600px"}`

Reply:
Hmm, it should be better `max-width` then, so that style-forced smaller sizes are possible. But I just tested, even with `max-width:600px` the style on small screen is broken since there seems to be no style rule that limits image sizes to the parent object size.

Another issue: On mid-size screens with collapsed navigation on the left size but visible TOC on the right size, the images are shown aside of the text instead of below the text as text and image fit into one line then. I guess an empty line in Markdown forces a line break in HTML.

However, if you think that 600px are really enough, I'd resize the png files themselves instead of limiting the size via styles, which also speeds up loading time. But still not sure since one cannot really read text of such small images. Yaah it's German language anyway but when I'm in mood to install an English language pack (or reinstall Englisch Windows from ground up) I'll redo them, probably with enhanced arrow graphics 😄.

Just recognised that the alt tags are all the same. And actually that section should be a collapsible box anyway.

Reply:
Okay images are automatically a bid smaller now due to the info box frame. An empty line before and after the images would make it look a bid better. @StephanStS what you think? https://dietpi.com/test/software/cloud/#bitwarden_rs

Reply:
That's fine.

Reply:
Sadly the empty line has no effect. It would required two manual line breaks or increased top margin or padding for the image. However, I'll leave it like that for now, it's really a minor thing.